pkg/webui/console/containers/device-onboarding-form/device-type-form-section/device-type-repository-form-section/device-card/device-card.js

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+
+import devicePlaceholder from '@assets/misc/end-device-placeholder.svg'
+
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { LORAWAN_VERSIONS, LORAWAN_PHY_VERSIONS } from '@console/lib/device-utils'
+
+import style from './device-card.styl'
+
+const m = defineMessages({
+  productWebsite: 'Product website',
+  dataSheet: 'Data sheet',
+  classA: 'Class A',
+  classB: 'Class B',
+  classC: 'Class C',
+})
+
+const getLorawanVersionLabel = ({ lorawan_version }) => {
+  const { label } = LORAWAN_VERSIONS.find(version => version.value === lorawan_version) || {}
+
+  return label
+}
+
+const getLorawanPhyVersionLabel = ({ lorawan_phy_version }) => {
+  const { label } =
+    LORAWAN_PHY_VERSIONS.find(version => version.value === lorawan_phy_version) || {}
+
+  return label
+}
```

Comment:
Let's move this to the device utils. We can use it on the end device overview as well.